packages/editor/src/components/char-count/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { withSelect } from '@wordpress/data';
+import { _x } from '@wordpress/i18n';
+import { count as wordCount } from '@wordpress/wordcount';
```

Comment:
The `wordCount` alias is a bit misleading, given it returns that characterCount when used in this file. It's minor, but it'd be good to either leave it as `count` or alias it as `charCount`.